packages/javascript-sdk/src/token-storage/session-storage.ts

```diff
@@ -18,19 +18,20 @@ abstract class SessionStorageWrapper {
   /**
    * Retrieve tokens.
    */
-  public static async get(clientId: string): Promise<Tokens> {
+  public static async get(clientId: string): Promise<Tokens | void> {
```

Comment:
Not important but we have some redundancy between `async` and using `Promise` since it inherently does this. 

Since we aren't awaiting this code at all, and we just return the `Promise` maybe we can just remove the `async` ?

Reply:
Addressed this by removing the returned Promises and keep the `async` function.